Gigs/Gigs/Controller/GigsTableViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  GigsTableViewController.swift
+//  Gigs
+//
+//  Created by Bradley Yin on 8/7/19.
+//  Copyright © 2019 bradleyyin. All rights reserved.
+//
+
+import UIKit
+
+class GigsTableViewController: UITableViewController {
+    
+    let gigController = GigController()
+
```

Comment:
I would *personally* prefer to see a single newline space between the following functions instead of butting them right up to each other.

Reply:
(butting up is acceptable though)